### ISSUE TITLE
[debian] 5.5.2 errors

### ISSUE THREAD

**Issue by picca** [vasole/pymca#594]
Hello armando

here an error in pymca 5.5.2
```
    File "/usr/lib/python3/dist-packages/PyMca5/Object3D/Object3DPlugins/ChimeraStack.py", line 72
      with h5py.File(filename, mode='r') as f
                                            ^
  SyntaxError: invalid syntax
 ```
llok at here for errors

https://salsa.debian.org/science-team/pymca/-/jobs/351017

**Comment by picca** [comment]
there is two of this error in this file

**Comment by vasole** [comment]
Thanks Fred, do you need a new release?

**Comment by picca** [comment]
I already uploaded it into Debian. For Debian there is no need, but for other maybe.

**Comment by vasole** [comment]
closed by #595
